### DISCUSSION TITLE
sliderendmiss@2x and slidertickmiss@2x should be automatically blank in each skin rather than all skins by default showing slider tick misses.

### DISCUSSION THREAD

**Discussion by cjwrecks** [ppy/osu#26408] [Ideas]
50% of the osu lazer update comments are all how to remove the slidertickmisses, no skins come with invisible versions of the elements meaning they always show slidertickmisses. It's great to add more functionality to skinning and more visuals for people who want it, but when you make the user experience for downloading and using skins harder and more time consuming- then something needs to change. 

In stable right now, if I see a skin online that I want and then download said skin, I can just open it and it will already be basically perfect, no 300's, etc. In lazer if I see a new skin I want and install it into osu and then see it has those slidertickmisses which for people like me and basically every player is just extra clutter we don't want to use ( and this is what 90% of players feel like ) I then have to delete the skin in lazer, open up stable, open the skin from stable, create 2 invisible .pngs', add them to the skin, then export from stable into lazer. 

All this could easily be fixed by either adding a toggle in the skin editor to show slidertickmisses, or by automatically adding blank pngs to skins without those elements already in them when imported considering 90% of the playbase will edit them out anyways.

I still think the option should be there but as something you add  yourself or something that comes downloaded in a skin like with particle effects on 100's and 50's in a skin like Xeltol's.

**Comment by paveldayo** [comment]
I agree!! For some time I thought I broke something while editing the skin, but no matter what I do, some weird lines was showing up on each slidertick miss.

I believe this feature by itself is useful for a lot of players, but I **personally** don't like it, and would like to be able to toggle it in the editor or settings (if it's not already an option and Im just blind)

**Comment by PuddlePumpkin** [comment]
Backwards compatibility with older skins does seem somewhat damaged by these being on by default.

Perhaps having to enable them in skin.ini would be a nice solution

**Comment by peppy** [comment]
In the next release we've toned down the visibility of these, so please see how things feel.

**Reply by cjwrecks** [reply]
I like them but I still think I'd rather it be defaulted to blank elements rather than always on because downloading new skins in for use in lazer means opening stable then dragging the new elements in, then exporting into lazer each and everytime when in stable i could just click and be done. Maybe add a "default skins to slidertickmisses off" toggle or something for people like me.

**Comment by PuddlePumpkin** [comment]
> I like them but I still think I'd rather it be defaulted to blank elements rather than always on because downloading new skins in for use in lazer means opening stable then dragging the new elements in, then exporting into lazer each and everytime when in stable i could just click and be done. Maybe add a "default skins to slidertickmisses off" toggle or something for people like me.

I wrote a simple batch script to add blank slider judgements to skins if anyone would find it helpful
https://github.com/PuddlePumpkin/OsuLazerAddBlankSliderJudgements
